sympy/concrete/tests/test_gosper.py

```diff
@@ -7,9 +7,10 @@
 
 
 def test_gosper_normal():
-    assert gosper_normal(4*n + 5, 2*(4*n + 1)*(2*n + 3), n) == \
+    eq = 4*n + 5, 2*(4*n + 1)*(2*n + 3)
```

Comment:
This will set the value of `eq` as `(4*n + 5, 2*(4*n + 1)*(2*n + 3))` and thus `gosper_normal(eq, n)` means `gosper_normal((4*n + 5, 2*(4*n + 1)*(2*n + 3)), n)` which will result in an error. I think it should be left as it is.